bot/exts/utils/clean.py

```diff
@@ -79,6 +149,10 @@ def predicate_regex(message: Message) -> bool:
             else:
                 return bool(re.search(regex.lower(), content.lower()))
 
+        def predicate_range(message: Message) -> bool:
+            """Check if message is older than message provided in after_message but younger than until_message."""
+            return message.created_at > after_message.created_at and message.created_at < until_message.created_at
```

Comment:
As the `until` clean is inclusive, meaning it deletes the message which is specified in the command, I think this should be inclusive as well, meaning have `<=` and `>=`.

Reply:
Sounds good, thanks, https://github.com/python-discord/bot/pull/1527/commits/708063c70bcea2cdeecb1eb66c703817e0195042